File: Clients/Android/app/src/main/java/Models/LearningResources/Dialogue.java
package Models.LearningResources;

import java.io.Serializable;

/**
 * Created by MARKAN on 19.05.2017.
 */
public class Dialogue implements Serializable {
    private Integer id;
    private String header;
    private String dialogueUkr;
    private String dialogueEng;
    private byte[] image;

    public Dialogue(Integer id, String header, String dialogueUkr, String dialogueEng, byte[] image) {
        this.id = id;
        this.header = header;
        this.dialogueUkr = dialogueUkr;
        this.dialogueEng = dialogueEng;
        this.image = image;
    }

    public Integer getId() {
        return id;
    }

    public byte[] getImage() {
        return image;
    }

    public String getDialogueEng() {
        return dialogueEng;
    }

    public String getDialogueUkr() {
        return dialogueUkr;
    }

    public String getHeader() {
        return header;
    }
}

File: Server/EasyUkr.WebApi/Models/RestModels/Dialogue.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace EasyUkr.WebApi.Models.RestModels
{
    public class Dialogue
    {
        public int Id { get; set; }
        public string Header { get; set; }
        public string DialogueUkr { get; set; }
        public string DialogueEng { get; set; }
    }
}